src/dotnet/CarbonAware.LocationSources.Azure/test/AzureLocationSourceTest.cs

```diff
@@ -10,34 +9,46 @@ namespace CarbonAware.LocationSources.Azure.Test;
 public class AzureLocationSourceTest
 {   
     [Test]
-    public async Task TestToGeoposition()
+    public void TestToGeopositionLocation()
     {
+        var logger = Mock.Of<ILogger<AzureLocationSource>>();
+
+        AzureLocationSource source = new AzureLocationSource(logger);
```

Comment:
This is unnecessary because it's taken care of by `SetupMockLocationSource().Object` call and ditto for tests below